.github/workflows/test.yml

```
@@ -0,0 +1,235 @@
+      - os: "macos-13"
+        python-version: "3.12"
+      - os: "macos-latest"
+        python-version: "3.11"
+      - os: "macos-latest"
+        python-version: "3.12"
+      - os: "windows-latest"
+        python-version: "3.11"
+      - os: "windows-latest"
+        python-version: "3.12"
+
+reusableSteps:
```

actionlint (1.7.4)

unexpected key "reusableSteps" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)